src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1540,5 +1540,12 @@ internal enum ErrorCode
         #endregion diagnostics introduced for `ref readonly`, `ref ternary` and `ref-like` features in C# 7.2
 
         ERR_PredefinedValueTupleTypeAmbiguous3 = 8356,
+
+        #region diagnostics introduced for C# 7.3
+        ERR_FeatureNotAvailableInVersion7_3 = 8360,
+        WRN_AttributesOnBackingFieldsNotAvailable = 8361,
```

Comment:
:memo: This is a warning (rather than an error) since such code currently produces a warning.
`[field: SomeAttribute] public int prop { get; set; }`